src/feature/projects/components/ProjectBanner.tsx

```diff
@@ -83,92 +83,105 @@ const SaleBox = styled(Box)`
   }
 `;
 
+// const CloseButton = styled(Button)<BoxProps>(
+//   css({
+//     display: ["block", "none"],
+//     padding: "0",
+//     width: "15px",
+//     height: "15px",
+//   })
+// );
+
+const CloseButton = styled(Button)`
+  padding: 0;
+  width: 15px;
+  height: 15px;
+  @media screen and (min-width: 1024px) {
+    display: none;
+  }
+`;
+
 export const ProjectBanner = ({ courseTitle }: Props) => {
-  const isXl = useMediaQuery("(min-width:1024px)");
-  const [isBannerOpen, setIsBannerOpen] = React.useState<boolean>(true);
+  const { isBannerOpen, setIsBannerOpen } = useLayoutContext();
+
+  if (isBannerOpen) {
```

Comment:
In questo modo continua ad inizializzare styled components e hooks. Perché non usarlo nel componente padre lo state. 

`isBannerOpen ? <ProjectBanner /> : null`